admin-events/agent-config.json

```diff
@@ -1,21 +1,27 @@
 {
```

Comment:
it appears that this config actually contains an example config, so the filename should probably be changed to something like `agent-config.json.example` that way it can be copied to the use in the actual config and users won't think it's a real config file